ui/src/app/views/concept-homepage/component.html

```diff
@@ -0,0 +1,81 @@
+      <button class="btn btn-secondary btn-search" (click)="searchButton()" [disabled]="getSearchDisabled">
+        Search
+        <span style="padding-left: 0.5rem; display: flex;" *ngIf="searchLoading"><span class="spinner spinner-sm"></span></span>
+      </button>
+      <span *ngIf="searchTermNotLongEnough" class="tooltip-content">Search must be at least three letters long</span>
+    </span>
+    <clr-checkbox class="standard-concepts-checkbox" [(clrChecked)]="standardConceptsOnly">
+      Standard concepts only
+    </clr-checkbox>
+  </div>
+  <div class="domain-box lower-box" *ngIf="!searching else chartBox">
+    <div *ngIf="loadingDomains else domains" class="spinner-container">
+      <span class="spinner spinner-lg"></span>
+    </div>
+    <ng-template #domains>
+      <div class="card item-card" *ngFor="let domain of conceptDomainList">
+        <div class="concept-name">
+          {{domain.name}}
+        </div>
+        <div class="concept-text">
+          <div>
+            <span class="concept-count-text">{{standardConceptsOnly ? (domain.standardConceptCount | number: '1.0-0') : (domain.allConceptCount | number: '1.0-0')}}</span>
+            <span class="concept-label-text">concepts</span>
+          </div>
+          in this domain.
+          <div style="margin-top: 0.5rem">
+            <span class="participant-count-text">{{domain.participantCount | number: '1.0-0'}}</span>
+            participants in this domain.
+          </div>
+        </div>
+        <div class="footer"><button class="btn btn-link browse-btn" (click)="browseDomain(domain)">Browse Domain</button></div>
+      </div>
+    </ng-template>
+  </div>
+  <ng-template #chartBox>
+    <div class="chart-box lower-box">
+      <div class="chart-selector-tabs">
+        <div *ngFor="let domain of conceptDomainList" class="chart-selector-tab">
+          <button (click)="selectDomain(domain.domain)" class="domain-selector-button btn btn-link nav-link nav-item" [class.active]="domain.domain === selectedDomain" [disabled]="searchLoading">
+            <div class="chart-selector-header">
+              {{domain.name}}
+            </div>
+            <!-- TODO: Make this the counts returned from search. -->
+            <div>{{domain.allConceptCount}}</div>
+          </button>
+        </div>
+      </div>
+      <!--<div class="filter-options">-->
+        <!--Filter by vocabulary:-->
+        <!--<clr-checkbox class="filter-checkbox" [clrChecked]="standardConceptsOnly">-->
+          <!--SNOMED (4011)-->
+        <!--</clr-checkbox>-->
+        <!--<clr-checkbox class="filter-checkbox" [clrChecked]="standardConceptsOnly">-->
+          <!--CPT (2951)-->
+        <!--</clr-checkbox>-->
+        <!--<clr-checkbox class="filter-checkbox" [clrChecked]="standardConceptsOnly">-->
+          <!--ICD10 (2112)-->
+        <!--</clr-checkbox>-->
+      <!--</div>-->
```

Comment:
Remove?

Reply:
I want this because it has all the styling and functionality worked out? Just non-functional yet.